Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -195,21 +197,33 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
 
         switch (this._type) {
         case WI.InlineSwatch.Type.Color:
+            let title = this._swatchTooltip;
             if (!this._readOnly) {
-                let title = WI.UIString("Click to select a color.");
+                title = WI.UIString("Click to select a color.");
                 if (this._allowChangingColorFormats())
                     title += "\n" + WI.UIString("Shift-click to switch color formats.");
                 if (InspectorFrontendHost.canPickColorFromScreen())
                     title += "\n" + WI.UIString("Option-click to pick color from screen.");
                 
                 this._swatchElement.title = title;
             }
+            if (title.length)
+                this._swatchElement.title = title;
             // fallthrough
 
         case WI.InlineSwatch.Type.Gradient:
             this._swatchInnerElement.style.background = value ? value.toString() : null;
             break;
 
+        case WI.InlineSwatch.Type.Variable:
+            if (!this._subType)
+                break;
+
+            let palette = this._colorPalette.find(palette => value.toString().slice(4, value.length - 1) == palette.colorSource || value.toString() === palette.colorSource);
```

Comment:
style: we always wrap the parameter of an arrow function in `(` `)`
style: we always use `===`